ruby/ruby-agreement.html

```diff
@@ -0,0 +1,38 @@
+<!doctype html>
+<html lang="en">
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width">
+<title>Agreement on HTML Ruby Markup</title>
+<style>
+body {
+	max-width: 50em;
+	margin: auto;
+	font-family: sans-serif;
+}
+</style>
+
+<h1>Agreement on HTML Ruby Markup</h1>
+
+<p>
+The W3C will specify extended HTML Ruby markup
+(based on <a href="https://github.com/whatwg/html/pull/6478">PR#6478</a>)
+under the <a href="https://www.w3.org/Consortium/Process/#rec-track">REC track</a>.
+This will be a derived document with some textual copying from <a href="https://html.spec.whatwg.org/multipage/">WHATWG HTML</a>
+and <a href="https://www.w3.org/TR/2014/NOTE-html-ruby-extensions-20140204/">W3C HTML Ruby Markup Extensions</a>,
+but the W3C and WHATWG mutually agree that publishing it is within the bounds of our <a href="https://www.w3.org/2019/04/WHATWG-W3C-MOU">MoU</a>.
+
+<p>
+W3C editors may offer pull requests to keep the HTML spec in sync,
+both technically and editorially,
+for the subset of features defined in both specs,
+so that readers of both specs can benefit from any improvements.
+Once the <a href="https://whatwg.org/working-mode#additions">criteria for addition</a> to WHATWG Living Standards are met,
+W3C editors may also offer pull requests to fully integrate the extended ruby markup definitions
+into WHATWG HTML.
+WHATWG HTML editors will review these contributions in good faith,
+and merge those they deem acceptable
+under the WHATWG <a href="https://whatwg.org/working-mode">working mode</a>.
```

Comment:
We'd like to propose a rewording:
```
In the event that W3C editors offer pull requests to fully integrate the extended ruby markup definitions into WHATWG HTML, once the <a href="https://whatwg.org/working-mode#additions">criteria for addition</a> to WHATWG Living Standards are met, WHATWG HTML editors will review these contributions in good faith, and merge those they deem acceptable under the WHATWG working mode.
```

Reply:
@plehegar I can't tell what this rewording is accomplishing, and I think it's harder to read. Can you explain what the problem is you're trying to solve?

Reply:
Thanks @fantasai for reaching out to discuss further. We found we were reading an ambiguity into "may" that could be resolved by indicating the [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) terminology, that MAY specifically means optionality rather than permission. 

I would support a global substitution of "MAY" for "may".

Reply:
See https://github.com/w3c/whatwg-coord/pull/15